contracts/exchange/NftEscrow.sol

```diff
@@ -85,7 +85,6 @@ contract NftEscrow is INftEscrow, EscrowBase, ERC1155HolderUpgradeable, ERC721Ho
         // We don't need to store how much was escrowed because we keep track of the order data in 
         // the orderbook.
         if (escrowedAmounts[_orderId] == 0) {
-            delete escrowedAmounts[_orderId];
```

Comment:
why did you remove this? Is it cheaper without this?

Reply:
Yes, because all deleting escrowedAmounts[_orderId] does is set the uint256 value to zero, and because of the if statement, it's already zero, so you don't get a refund for it (or you've already been refunded and you don't get another refund for it). I ran a gas test to verify it.

Reply:
Sounds good.